app/templates/components/author-assertions.hbs

```diff
@@ -1,6 +1,6 @@
 {{#if this.shouldShowSloanIcons}}
     <div class='author-assertions-header-label'>
-        {{~t 'components.author-assertions.header_label'~}}
+        {{~t 'components.preprint-form-header.name.Assertions'~}}
```

Comment:
Translations are grouped by component. Is there any specific reason why this changed despite the translation being used in `author-assertions`?